dbt/models/iasworld/columns.md

```diff
@@ -862,12 +851,6 @@ Relative location
 Flag to indicate suppress value
 {% enddocs %}
 
-## taxyr
-
-{% docs column_taxyr %}
-Tax year
-{% enddocs %}
-
```

Comment:
Dropping these simply because the definitions inside `shared_columns.md` will take their place.